src/components/home/VolunteerOpportunities.tsx

```diff
@@ -15,10 +15,10 @@ const VolunteerOpportunities = () => {
         free to show up! We'd love all the help we can get!
       </p>
       <div className="flex w-full flex-row py-2 sm:py-10">
-        <div className="hidden items-center justify-center sm:flex">
+        <div className="hidden justify-center sm:flex md:mx-8 md:mt-24 md:h-full md:w-3/5 lg:my-10 lg:w-full">
           <Photo
             image={VolunteerPhoto}
-            width={"w-1/2 lg: w-3/4"}
+            width=""
```

Comment:
Instead of having a width prop
In the `Photo` component, add a default prop for width as an empty string (so that you don't need to pass in  this prop when calling the function).

Also are you sure that you want to get rid of the width for this particular photo?

Reply:
whenever I tried using width, it would make the photo smaller than I would like